docs/Architecture.md

```diff
@@ -10,7 +10,7 @@ Nussknacker consists of three parts:
 
 
 ##Engine
-Engine consists of various modules that enable creation of processes building blocks in UI and interpretation of process diagrams. Engines implement `ProcessManagerProvider`.
+Engine consists of various modules that enable creation of processes building blocks in UI and interpretation of process diagrams. Engines implement `DeploymentManagerProvider`.
```

Comment:
We should change this chapter. It should be sth like "scenario deployment". We should write that "Scenario is deployed on engine (like Flink) using DeploymentManager"